src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -33,6 +34,12 @@ public SkillController(SkillRepository skillRepository) {
                 .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
+    @GetMapping("/search/{name}")
+    public ResponseEntity<List>getName(@PathVariable(value = "name") String skillName){
+        List<Skill> skillOptional = skillRepository.findByNameContaining(skillName);
```

Comment:
is the list returned Optional?
We talked about clean code so this name does not make sense. I'd rather do something like:

List<Skill> foundSkills = skillRepository.findByNameContaining(skillName); 
or 
List<Skill> skillWithNameCoincidences = skillRepository.findByNameContaining(skillName);

Reply:
I agree, I changed the name as we discussed it in the clean code meeting.